### PR TITLE
Adding PATH parser for windows

### PR DESCRIPTION
**Pull request by roigcarlo**
**📝 Description**
Adds the PATH directories to the search table of shared libs for python >= 3.8 in windows.

This is a quick fix until a better solution is found. Should fix MKL / PARDISO / FEAST / HDF5, etc... problems in win with 3.8

**🆕 Changelog**
- Adding PATH entries to the shared lib lookup locations for python 3.8

### CODE REVIEW

**Review by loumalouomega — APPROVED** [review]

**Review by AlejandroCornejo — CHANGES_REQUESTED** [review]
Gonna try it now

**Comment by matekelemen** [comment]
oh god ...
what happened?

**Comment by AlejandroCornejo** [comment]
> oh god ... what happened?

Windows peasants like me could not use MKL hehehe ;)

**Comment by roigcarlo** [comment]
> oh god ... what happened?

https://docs.python.org/3/whatsnew/3.8.html#bpo-36085-whatsnew

TL;DR, if you have your DLL paths in PATH they are not loaded anymore.

I think it has to do with the fact that both bin and DLL were being loaded from path and is somehow exploitable, but haven't dig into the details. 

Not sure yet up to which point it can be fixed by us alone. In the case of MKL for example the problem is that, even if we copy the DLL into Kratos folder in the install stage, the dependencies of the dependencies are not copied, and since the direct dependency is a entry-point that dynamically loads the correct lib with `LoadLibrary` directly from the code is impossible (or I don't know how to, not gonna parse the strings of the binary....) catch it at that stage.

So that's the better I can come up with for now :S

**Review by AlejandroCornejo — APPROVED** [review]
It's working!! :)

Back to the future again :D

**Comment by AlejandroCornejo** [comment]
@lagoncalvesjr FYI

**Comment by AlejandroCornejo** [comment]
BTW we should update the README of the LinearSolvers then

**Comment by AlejandroCornejo** [comment]
@roigcarlo should we merge?

**Comment by roigcarlo** [comment]
Yup, if no one else sees any problem it can go in.

I promise we will look for a better solution
